libparsec/crates/client/src/workspace/merge.rs

```diff
@@ -263,27 +255,24 @@ pub(super) fn merge_local_folder_manifest(
     // /!\ Extra attention should be paid here if we want to add new fields
     // /!\ with their own sync logic, as this optimization may shadow them!
 
-    let merged_need_sync = merged_children != remote.children || merged_parent != remote.parent;
-    let updated = if merged_need_sync {
+    let merge_need_sync =
+        merged_children != local_from_remote.children || merged_parent != local_from_remote.parent;
+    let merge_update = if merge_need_sync {
         timestamp
     } else {
-        remote.updated
+        local_from_remote.updated
     };
-    // TODO: prevent sync pattern is not handled here !
-    let merged_manifest = LocalFolderManifest {
-        need_sync: merged_need_sync,
-        parent: merged_parent,
+
+    let manifest = LocalFolderManifest {
         children: merged_children,
-        updated,
-        base: remote,
-        // TODO: confinement points support not implemented yet !
-        local_confinement_points: HashSet::new(),
-        // TODO: confinement points support not implemented yet !
-        remote_confinement_points: HashSet::new(),
+        parent: merged_parent,
+        need_sync: merge_need_sync,
+        updated: merge_update,
         speculative: false,
+        ..local_from_remote
```

Comment:
Again, sorry to insist by I'd rather have an explicit code about what goes in `merged_manifest` instead of `..local_from remote` which tends to implicitly resolve newly added field to whatever is in `local_from_remote`

Reply:
done